smb/src/main/java/ch/cyberduck/core/smb/SMBCopyFeature.java

```diff
@@ -96,16 +82,17 @@ private Path copyFile(Path source, Path target, TransferStatus status, Connectio
             sourceFile.remoteCopyTo(targetFile);
         }
         catch(TransportException | BufferException e) {
+            throw new BackgroundException(e);
```

Comment:
Replace with parameters `"Cannot copy {0}", e, source`

```suggestion
            throw new BackgroundException("Cannot copy {0}", e, source);
```